Modules/Front/Resources/views/code/laravelExperiments.blade.php

```diff
@@ -0,0 +1,16 @@
+&lt;?php
+
+use Abrouter\Client\Client;
+
+class ExampleController
+{
+    public function __invoke(Client $client)
+    {
+        $buttonColor = $client->experiments()->run(uniqid(), '<?=request('id')?>');
```

Comment:
Please, dont use the request() directly here. Pass it from the controller. Additionally, it will be better to display it as {{$requestId}} - with blade